unison-src/transcripts-round-trip/main.output.md

```diff
@@ -241,40 +241,47 @@ f x = let
 Regression test for https://github.com/unisonweb/unison/issues/2224
 
 ```unison
-f : [a] -> a
+f : [()] -> ()
 f xs = match xs with
   x +: (x' +: rest) -> x
+  _ -> ()
 
-g : [a] -> a
+g : [()] -> ()
 g xs = match xs with
-  (rest :+ x') :+ x -> x
+  (rest :+ x') :+ x -> ()
+  _ -> ()
 
-h : [[a]] -> a
+h : [[()]] -> ()
 h xs = match xs with
   (rest :+ (rest' :+ x)) -> x
+  _ -> ()
 ```
 
 ```ucm
 .> add
 
   ⍟ I've added these definitions:
   
-    f : [a] -> a
-    g : [a] -> a
-    h : [[a]] -> a
+    f : [()] -> ()
+    g : [()] -> ()
+    h : [[()]] -> ()
 
 .> edit f g
 
   ☝️
   
   I added these definitions to the top of
-  /Users/runar/work/unison/scratch.u
+  /home/traveler/code/haskell/unison/pattern-match-coverage/scratch.u
```

Comment:
Unrelated, but I wonder if we could get UCM to run in the directory with the transcript @ChrisPenner

Reply:
I think this file just shouldn't be checked in. CI just runs the transcript to make sure it passes, and that's sufficient.